.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       CGO_ENABLED: 0
     steps:
       - checkout
-      - run: gometalinter ./...
+      - run: make lint
```

Comment:
That image `supinf/gometalinter` doesn't have make installed :(

Reply:
boo 🙈